doc/setup/quickstart_cmake/index.md

```diff
@@ -104,8 +105,7 @@ To compile it we first create a `CMakeLists.txt` file in the `source` directory:
 cmake_minimum_required (VERSION 3.4)
 project (seqan3_tutorial CXX)
 
-set(SeqAn3_DIR "${CMAKE_SOURCE_DIR}/../seqan3/build_system")
-find_package (SeqAn3 REQUIRED)
+add_subdirectory ("${CMAKE_SOURCE_DIR}/../seqan3" seqan3-build)
```

Comment:
So now, I have to call add_subdirectory instead of find_package in my CMakeProject? That would not be my favorite style for adding SeqAn into my project, even if it might safe a line.